test/testdata/namer/type_alias_not_on_T.rb.symbol-table-raw.exp

```diff
@@ -0,0 +1,28 @@
+class <C <U <root>>> < <C <U Object>> ()
+  class <S <C <U <root>>> $1>[<C <U <AttachedClass>>>] < <S <C <U Object>> $1> ()
+    method <S <C <U <root>>> $1>#<N <U <static-init>> $CENSORED> (<blk>) @ Loc {file=test/testdata/namer/type_alias_not_on_T.rb start=2:1 end=20:4}
+      argument <blk><block> @ Loc {file=test/testdata/namer/type_alias_not_on_T.rb start=??? end=???}
+  module <C <U A>> < <C <U Sorbet>>::<C <U Private>>::<C <U Static>>::<C <U ImplicitModuleSuperclass>> () @ Loc {file=test/testdata/namer/type_alias_not_on_T.rb start=2:1 end=2:9}
+    static-field <C <U A>>::<C <U NotActuallyATypeAlias>> -> <todo sym> @ Loc {file=test/testdata/namer/type_alias_not_on_T.rb start=9:3 end=9:24}
```

Comment:
This `static-field` is the key part. Prior to these changes, this test yielded `static-field-type-alias` here.

Reply:
I think the same test could have been written with `symbol-table.exp` (not raw) since the test doesn't rely on raw names like `<C <U Private>>` to catch regressions. Even the test below which shows the error saying that this thing is not a type alias is sufficient. Anyways, changing this would not really improve the PR so I'm just commenting this for future reference.

Reply:
Good to know, thanks!

I switched it, since I edited the commit to lint, anyway.